optimum/amd/ryzenai/modeling.py

```diff
@@ -231,7 +226,7 @@ def _generate_regular_names_for_filename(filename: str):
         return [filename, f"{name}_quantized.{extension}", f"{name}_optimized.{extension}"]
 
     @classmethod
-    def _from_pretrained(
+    def _load_model_and_processors(
```

Comment:
Maybe we discussed this already but why the renaming?

Reply:
This method is not actually renamed, but a new method is created. I could reuse the `_load_model_and_processors` in `_from_pretrained` of `CausalLM` class. The UI is making it look like its renamed.